index.js

```diff
@@ -138,7 +146,7 @@ function runCumulusTask(taskFunction, cumulusMessage, context, callback) {
     );
   }
   else {
-    const promisedRemoteEvent = loadRemoteEvent(cumulusMessage);
+    const promisedRemoteEvent = loadRemoteEvent(cumulusMessage, schemaLocation);
```

Comment:
I am confused about where `schemaLocation` comes from? I think it should be an optional param to `runCumulusTask`.